.github/workflows/rspec.yml

```diff
@@ -0,0 +1,32 @@
+name: RSpec
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "0 21 * * 6"
+
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        mongodb-version: ["4.0", "4.2", "4.4", "5.0", "6.0"]
```

Comment:
Is this list of mongo versions one that you pulled from somewhere else? I think its probably overkill here and a bit wasteful. If the idea of this is to be a canary for upstream changes that break errbit, maybe we want to run this job on only the 'latest' mongo somehow..

Reply:
I take it from Circle CI config. And add 6.0. Circle CI uses different mongodb images. Their own. Right now, they are deprecated. I don’t want to change or update circle ci config. We should remove it. But for now, it has some external test coverage. I will take care about this later.

We can remove older versions of mongodb. But, lets discuss this in another PR. Plus, I run errbit with mongodb 4.0.